conventions/documentation/toast-notifications.md

```diff
@@ -0,0 +1,41 @@
+---
+parent: Conventions
+---
+
+# Toast notifications
+
+Toast notifications can be used to convey important messages or updates to the user regardless of where they are in the console. They should be used sparingly in the OpenShift console.
+
+## Guidelines
+### When to use
+
+* To convey console related alerts that appear as a result of the user taking an action in the UI
+* To convey status about longer running actions like:
+  * A file being uploaded to Topology
+  * Something being updated
+  * Data being migrated
+* To provide status messaging for a user-initiated process that is asynchronous
+* To show an error related actions that would have otherwise been conveyed as a toast
+
```

Comment:
Instead of "To show an error related actions that would have otherwise been conveyed as a toast" , what if we said "To show error messages related to actions taken in the UI" ?

Reply:
That sounds good to me! I'll update that.